tests/integration/ethernet_mtu_test.py

```diff
@@ -43,6 +46,9 @@ def test_decrease_iface_mtu():
     eth1_desired_state = desired_state[INTERFACES][0]
     eth1_desired_state['state'] = 'up'
     eth1_desired_state['mtu'] = 1000
+    eth1_desired_state['ipv6']['enabled'] = False
+    eth1_desired_state['ipv6']['address'] = []
+    # IPv6 requires MTU >= 1280 (RFC 2460)
```

Comment:
Can you add the comment before the ipv6 editing and explain why in this case we need to specify ipv6 `enabled=False`?
Is is also the default, so why even bother specifying it? I am missing something here.

Also, perhaps it is worth having now a test with `mtu=1280` for the "decrease" general case and the `mtu=1000` for the ipv6 case (only the latter needs explanation).

Reply:
It's kernel QE's job to test whether IPv6 got disabled when MTU < 1280, not us.

Reply:
We need to check possible scenarios of input and see that nmstate behaves correctly.
Anyway, you have adjusted an existing test, so my main comment is about that. Adding another scenario is not a must, we can add it later.